documentation/docs/connectors/binance.md

```diff
@@ -30,11 +30,15 @@ Because Binance is a centralized exchange, you will need to generate and provide
 <br />
 5 - Now you have created an API key. Please note that to trade on Binance using Hummingbot, **Enable Trading** must be selected.
 
+!!! warning "API key permissions"
+    For general use of Hummingbot, we recommend "trade only" enabled API keys and suggest not using "transfer" or "withdraw" enabled API keys.
```

Comment:
We recommend using only "trade" enabled API keys; enabling "withdraw" is unnecessary for current Hummingbot strategies.

Reply:
Rephrased. Thanks!